docs/source/ipython_config.py

```diff
@@ -0,0 +1,2 @@
+
+c.InteractiveShellApp.extensions = ['autotime']
```

Comment:
I'm guessing this will also be deleted before merge?

Reply:
Yes!